services/course-material/src/components/ContentRenderer/moocfi/AsideWithImageBlock.tsx

```
@@ -0,0 +1,96 @@
+          font-family: ${primaryFont};
+          margin-bottom: 1.125rem;
+          font-weight: 550;
+          font-size: 1.5rem;
+          color: #1a2333;
+        }
+      `}
+    >
+      <InnerBlocks parentBlockProps={props} />
+      <div>
+        <h4
+          dangerouslySetInnerHTML={{
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)